modality-probe-cli/src/log/mod.rs

```diff
@@ -82,6 +86,10 @@ pub struct Log {
 
     #[structopt(short, long, help = FORMAT_DOC)]
     pub format: Option<String>,
+    /// Filter a whole graph down to the radius around a specific
+    /// event.
```

Comment:
I think it'd be helpful here or in the about/help to mention what the units of `--radius` mean.

Reply:
ah, I see, it's now about a specific event.

Reply:
Can you add some guidance or examples somewhere in the README or here in this help about how to use `--radius`?

Reply:
Yeah, I intend to make a docs pass this morning.